tests/components/hassio/test_init.py

```diff
@@ -536,14 +527,14 @@ async def test_service_calls(
     )
     await hass.async_block_till_done()
 
-    assert aioclient_mock.call_count + len(supervisor_client.mock_calls) == 24
+    assert aioclient_mock.call_count + len(supervisor_client.mock_calls) == 25
```

Comment:
For reviewers - Previously tests were mocking the `HassIO.is_connected` method rather then the url `supervisor/ping` so that call was not included in this count. Now that method in the library is being mocked just like all the others so the count increases by one.